content/articles/mobx.md

```diff
@@ -0,0 +1,355 @@
+Title: A MobX introduction and case study
+Date: 2016-10-26
+Short_summary: Getting started with MobX and a real example
+Category: Dev
+Authors: Vincent
+
+*Reading time: ~15 minutes.*
+
+[MobX](https://mobxjs.github.io/mobx/) is a state management library for JavaScript frontend application. This article is an introduction to it by example as well as showing how it works in a real app with TypeScript and is based on a talk I gave at the Osaka Web designers and developers meetup recently.
```

Comment:
quite long, I'd try to split into 2 sentences

> is an introduction to it

you can verb, e.g. "This article introduces it with examples."